src/pages/Settings/Settings.tsx

```diff
@@ -905,277 +1071,354 @@ const Settings = () => {
 	};
 
 	return (
-		<>
-			<div className="px-6 sm:px-12 w-full">
-				{userData && (
-					<>
-						<H1 heading={t('common.navItemSettings')} />
-						<PageDescription description={t('pageSettings.description')} />
-
-						<div className="my-2 py-2">
-							<H2 heading={t('pageSettings.title.language')} />
-							<div className="relative inline-block min-w-36 text-gray-700">
-								<div className="relative">
+		<div className="px-4 sm:px-8 lg:px-12 pt-10 pb-20 w-full max-w-[1064px] mx-auto">
```

Comment:
```suggestion
		<div className="px-6 sm:px-12 w-full">
```
let keep it the same with the other pages, 
about the `max-w-[1064px] mx-auto`may add it on layout.jsx